docs/dev/building-on-zksync/contracts/differences-with-ethereum.md

```diff
@@ -97,29 +98,29 @@ export function createAddress(sender: Address, senderNonce: BigNumberish) {
 
 ### Differences in `block.timestamp` and `block.number`
 
-These variables behave differently from L1. Read more [about blocks in zkSync](../../developer-guides/transactions/blocks.md#blocks-in-zksync-era).
+You can find more information about blocks in zkSync, including the differences in block.timestamp and block.number, [in this link.](../../developer-guides/transactions/blocks.md#blocks-in-zksync-era).
 
 ### Differences in `COINBASE` `DIFFICULTY` `BASEFEE` 
+In zkSync, there are some differences in the behavior of certain variables compared to L1.
 
-- `COINBASE` returns our Bootloader contract address, `0x0000000000000000000000000000000000008001` 
-- `DIFFICULTY` returns a constant value of `2500000000000000` 
-- `BASEFEE` is not a constant, but is defined by the fee model. Most of the time it is 0.25 gwei, but under very high L1 gas prices it may rise.
+- The `COINBASE` variable in zkSync returns the address of the Bootloader contract, which is `0x0000000000000000000000000000000000008001`.
 
-### Use of `codesize`/`codecopy` and `calldata` in the deploy code
+- The `DIFFICULTY` variable returns a constant value of `2500000000000000`.
 
-`Calldata` in the constructor is not empty as on the EVM, on zkEVM calldata contains constructor arguments.
```

Comment:
Did you mean to delete all these lines?

Reply:
This isn't what my local branch looks like, will push again